lib/xcodeproj/project/object/file_system_synchronized_exception_set.rb

```diff
@@ -0,0 +1,62 @@
+require 'xcodeproj/project/object_attributes'
+require 'xcodeproj/project/object/helpers/groupable_helper'
+
+module Xcodeproj
+  class Project
+    module Object
+      # This class represents a file system synchronized build file exception set.
+      class PBXFileSystemSynchronizedBuildFileExceptionSet < AbstractObject
+        # @return [AbstractTarget] The target to which this exception set applies.
+        #
+        has_one :target, AbstractTarget
+
+        # @return [Array<String>] The list of files in the group that are excluded from the target.
+        #
+        attribute :membership_exceptions, Array
+
+        # @return [Array<String>] The list of public headers.
+        #
+        attribute :public_headers, Array
+
+        # @return [Array<String>] The list of private headers.
+        #
+        attribute :private_headers, Array
+
+        # @return [Hash] The files with specific compiler flags.
+        #
+        attribute :additional_compiler_flags_by_relative_path, Hash
+
+        # @return [Hash] The files with specific attributes.
+        #
+        attribute :attributes_by_relative_path, Hash
+
+        # @return [Hash] The files with a platform filter.
+        #
+        attribute :platform_filters_by_relative_path, Hash
+
+        def display_name
+          "Exceptions for \"#{GroupableHelper.parent(self).display_name}\" folder in \"#{target.name}\" target"
```

Comment:
is there a reason you have this explicit string here vs what Xcode itself generates here: `PBXFileSystemSynchronizedBuildFileExceptionSet `

Xcode replaces this string with that when editing the project file, so we're seeing a back-and-forth diff of this comment being edited by both parties.

Reply:
I checked in Xcode 16.2RC and this string format is what is used on my projects still.

Reply:
https://github.com/CocoaPods/Xcodeproj/pull/1000

oh I'm on 16.1, let me try 16.2 real quick to verify

Reply:
hmm Xcode 16.2 RC still reverts this comment:
1. we're calling fastlane `increment_version_number_in_xcodeproj`, which just does `project = Xcodeproj::Project.open`, edits a build setting, then does `project.save`
2. this custom comment then gets added
3. move a package/group/file in Xcode that is tracked in the project file
4. the comment now becomes `PBXFileSystemSynchronizedBuildFileExceptionSet`